site-src/guides/rollout.md

```diff
@@ -0,0 +1,197 @@
+data:
+  configmap.yaml: |
+    vLLMLoRAConfig:
+      name: vllm-llama3-8b-instruct-adapters
+      port: 8000
+      defaultBaseModel: meta-llama/Llama-3.1-8B-Instruct
+      ensureExist:
+        models:
+        - id: food-review-1
+          source: Kawon/llama3.1-food-finetune_v14_r8
+        - id: food-review-2
+          source: Kawon/llama3.1-food-finetune_v14_r8
+```
+
+The new adapter version is applied to the model servers live, without requiring a restart.
+
+
+### Direct traffic to the new adapter version
+
+Modify the InferenceModel to configure a canary rollout with traffic splitting. In this example, 10% of traffic for food-review model will be sent to the new ***food-review-2*** adapter.
+
+
+```bash
+kubectl edit inferencemodel food-review
+```
+
+Change the targetModels list in InferenceModel to match the following:
+
+
+```yaml
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: food-review
+spec:
+  modelName: food-review
+  criticality: Standard
+  poolRef:
+    name: vllm-llama3-8b-instruct
+  targetModels:
+  - name: food-review-1
+    weight: 90
+  - name: food-review-2
+    weight: 10
+```
+
+The above configuration means one in every ten requests should be sent to the new version. Try it out:
+
+1. Get the gateway IP:
+```bash
+IP=$(kubectl get gateway/inference-gateway -o jsonpath='{.status.addresses[0].value}'); PORT=80
+```
+
+2. Send a few requests as follows:
+```bash
+curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
+"model": "food-review",
+"prompt": "Write as if you were a critic: San Francisco",
+"max_tokens": 100,
+"temperature": 0
+}'
+```
+
+### Finish the rollout
+
+
+Modify the InferenceModel to direct 100% of the traffic to the latest version of the adapter.
+
+```yaml
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: food-review
+spec:
+  modelName: food-review
+  criticality: Standard
+  poolRef:
+    name: vllm-llama3-8b-instruct
+  targetModels:
+  - name: food-review-2
+    weight: 100
+```
+
+Unload the older versions from the servers by updating the LoRA syncer ConfigMap to list the older version under the `ensureNotExist` list:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vllm-llama3-8b-instruct-adapters
+data:
+  configmap.yaml: |
+    vLLMLoRAConfig:
+      name: vllm-llama3-8b-instruct-adapters
+      port: 8000
+      defaultBaseModel: meta-llama/Llama-3.1-8B-Instruct
+      ensureExist:
+        models:
+        - id: food-review-2
+          source: Kawon/llama3.1-food-finetune_v14_r8
+      ensureNotExist:
+        models:
+        - id: food-review-1
+          source: Kawon/llama3.1-food-finetune_v14_r8
+```
+
+With this, all requests should be served by the new adapter version.
+
+## **node-update-rollout**
+Node update roll outs safely migrate inference workloads to new node hardware or accelerator configurations. 
```

Comment:
Node rollout is confusing because it can mean updating the kubernetes node versions. Pls update the doc and be specific that this is about rolling out a new InferencePool. 

While upgrading nodes can be a reason to do an InferencePool rollout, there can be other reasons (trying a new accerlerator, a new model server , etc.)